src/runtime_src/tools/xclbinutil/XclBinUtilMain.cxx

```diff
@@ -55,24 +55,23 @@ void drcCheckFiles(const std::vector<std::string> & _inputFiles,
 {
    std::set<std::string> normalizedInputFiles;
 
-   for( auto file : _inputFiles) {
-     if ( !std::filesystem::exists(file)) {
+   for (const auto& file : _inputFiles) {
+     if (!fs::exists(file)) {
        std::string errMsg = "ERROR: The following input file does not exist: " + file;
        throw std::runtime_error(errMsg);
      }
-     std::filesystem::path filePath(file);
+     fs::path filePath(file);
      normalizedInputFiles.insert(canonical(filePath).string());
    }
 
    std::vector<std::string> normalizedOutputFiles;
    for ( auto file : _outputFiles) {
-     if ( std::filesystem::exists(file)) {
+     if ( fs::exists(file)) {
        if (_bForce == false) {
          std::string errMsg = "ERROR: The following output file already exists on disk (use the force option to overwrite): " + file;
          throw std::runtime_error(errMsg);
        } else {
-         std::filesystem::path filePath(file);
-         normalizedOutputFiles.push_back(canonical(filePath).string());
+         normalizedOutputFiles.push_back(fs::canonical(file).string());
        }
      }
```

Comment:
Early exit / continue and  no curly braces around single statements.

Reply:
Done